cloudformationTemplates/startChatContactAPI/README.md

```diff
@@ -1,83 +1,173 @@
+                            "customerName": customerName
+                        }),
+                        contactFlowId: contactFlowId,
+                        instanceId: instanceId
+                    },successHandler, failureHandler);
+
+                }
+            });
+        });
+
+        function successHandler(chatSession) {
+            console.log("success!");
+            $('#section-chat').fadeIn(400);
+
+            chatSession.onChatDisconnected(function(data) {
+                $('#section-chat').hide('slide');
+            });
+        }
+
+        function failureHandler(error) {
+            console.log("There was an error: ");
+            console.log(error);
+        }
+    </script>
     ```
+5. Type your name and click on start chat. And then Go to your connect contact center https://<YourContactCenterName>.awsapps.com/connect/home and click on "Test Chat". Change your agent status as available to test the chat workflow
+
+    ![Test Chat](images/testChat.png) 
+
+6. That's it! Now you should be able to test AWS Connect Chat experience.
 
-## Creating your own Chat UX
+## Getting Started with a Custom Widget
 
 If you want to build your own Chat widget instead of using our prebuilt one, follow these step by step directions. If you want to see a complete example of how to use [ChatJS](https://github.com/amazon-connect/amazon-connect-chat-js), look at the  `customBuildIndex.html` file.
 
 1) First, you need deploy the backend API as instructed above.
 2) Once your stack is deployed, go to the [API Gateway console](https://console.aws.amazon.com/apigateway/home), select the API, go to the Stages menu item, and select the `Prod` stage. You will then see the `Invoke URL`. This is the URL you will invoke to start the chat.
-![api gateway url diagram](images/apiGatewayUrl.png)
-3) Gather the contact flow ID and contact flow ID you want to use. If you don't have a contact flow, use the `Sample inbound flow` one in your instance that is created by default. You can find these IDs when viewing a contact flow. For example, if the arn for your flow is `arn:aws:connect:us-west-2:123456789012:instance/11111111-1111-1111-1111-111111111111/contact-flow/22222222-2222-2222-2222-222222222222`, the instance ID is `11111111-1111-1111-1111-111111111111` and the contact flow ID is `22222222-2222-2222-2222-222222222222`
-![contact flow ids diagram](images/contactFlowIds.png)
-4) Download the [compiled ChatJS source](https://github.com/amazon-connect/amazon-connect-chatjs/blob/master/dist/amazon-connect-chat.js) and save it locally.
-5) Create an `index.html` file in the same directory as the ChatJS source
-6) Set up your `index.html` file by importing ChatJS and creating the html elements you will use. These include an entry for the customer's name, a chat content input field, an area for the chat transcript to show up, and buttons to start the chat, get the transcript, send a typing event, and end the chat. 
+    <picture ?>
```

Comment:
Why did you remove the api gateway pic?

Reply:
It exists in the pre built widget bullet #3